apps/prairielearn/public/stylesheets/local.css

```diff
@@ -423,3 +423,7 @@ small .user-output-invalid {
 .fixed-table-toolbar div.pagination-detail {
   margin: 0 1em 0 0 !important;
 }
+
+.dropdown-toggle-no-caret::after {
+  display: none;
+}
```

Comment:
Bootstrap doesn't seem to have any built-in solution for what I wanted: a button with a custom icon that opened a dropdown.

Reply:
I ended up removing this in db7ac29.